components/geometry/lib.rs

```diff
@@ -6,7 +6,8 @@ extern crate app_units;
 extern crate euclid;
 extern crate malloc_size_of;
 extern crate style_traits;
-#[macro_use] extern crate malloc_size_of_derive;
+#[macro_use]
+extern crate malloc_size_of_derive;
```

Comment:
I'm not sure why this got past test-tidy previously, but please move this crate above style_traits.

Reply:
Done.